src/AutoUI/Actions/Tags.tsx

```diff
@@ -25,7 +27,18 @@ export const Tags = <T extends AutoUIBaseResource<T>>({
 }: TagsProps<T>) => {
 	const { t } = useTranslation();
 
-	const { sdk } = autouiContext;
+	const { sdk, internalPineFilter, checkedState } = autouiContext;
+
+	const { data: items, isLoading } = useRequest(
+		async () => {
+			if (checkedState === 'all' && sdk?.tags && 'getAll' in sdk.tags) {
+				return await sdk.tags.getAll(internalPineFilter);
```

Comment:
Should we only do this when in server side pagination mode?
The reasoning being that if we are on client side mode, then we already have all the data, and refetching the tags would only delay the modal (especially for big fleets).